doc/sphinx/practical-tools/utilities.rst

```diff
@@ -81,71 +84,71 @@ be copied or moved.
 We encourage you to use a source code control system for any non-trivial
 project because it makes it easy to track the history of your changes.
 `git <https://git-scm.com/>`_ is the system most used by Coq projects.
-Typically, each package has its own git repository.
-
-**Single directory projects.**  If your project has a single file (and
-depends only on packages in the Coq installation), put it in any directory
-you like and run Coq from that directory.  If your project has multiple files,
-then you need to compile them in an order that matches their dependencies.
-If B.v depends on A.v, then you should compile A.v before B.v.  You can do this
-with :n:`coqc A.v` followed by :n:`coqc B.v`, but you may find it tedious to manage
-the dependencies.  You may find it easier to treat your project as a
-multi-directory project.
-
-**Multi-directory projects.**  For these, we recommend using `_CoqProject` files
-to define one or more packages for your project.  With that, you can automatically
-generate a makefile that matches the dependencies.
-A :gdef:`package` is a group of files in a top directory and its subdirectories
-that's built and installed as a unit, just like the opam packages mentioned above.
-`_CoqProject` must be in the top directory.
+Typically, each project has its own git repository.
```

Comment:
Don't know if it might be worth mentioning what git is.

Reply:
The 3 immediately proceeding lines are:

```
We encourage you to use a source code control system for any non-trivial
project because it makes it easy to track the history of your changes.
`git <https://git-scm.com/>`_ is the system most used by Coq projects.
```

What more do you want?

Reply:
Oh right missed that.